### PR TITLE
Move loadflow, sensi and sa providers to their corresponding servers

### PR DESCRIPTION
**Pull request by ayolab**

### CODE REVIEW

File: src/main/java/org/gridsuite/study/server/RestTemplateConfig.java [hunk 1]
```diff
@@ -29,7 +30,7 @@ public class RestTemplateConfig {
 
     @Bean
     public RestTemplate restTemplate() {
-        final RestTemplate restTemplate = new RestTemplate();
+        final RestTemplate restTemplate = new RestTemplate(new HttpComponentsClientHttpRequestFactory());
```
[the remark this answers is not in the document]
Comment (ayolab) [reply src/main/java/org/gridsuite/study/server/RestTemplateConfig.java:33]: the Apache HttpComponents dependency was missing, Fixed

File: pom.xml [hunk 2]
```diff
@@ -211,6 +211,11 @@
             <artifactId>spring-cloud-stream-binder-rabbit</artifactId>
             <scope>runtime</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents.client5</groupId>
+            <artifactId>httpclient5</artifactId>
+            <version>5.3</version>
```
Comment (SlimaneAmar) [inline pom.xml:217]: Move version in a property and use it in the dependencyManagement

**Review by dbraquart — CHANGES_REQUESTED** [review]
code: only a remark
tests: ok

**Review by dbraquart — APPROVED** [review]
code: ok
tests: ok

**Review by SlimaneAmar — APPROVED** [review]

**Pull request merged**